htdocs/fourn/commande/index.php

```diff
@@ -162,7 +162,8 @@
 $sql.= ", ".MAIN_DB_PREFIX."commande_fournisseur as cf";
 if (!$user->rights->societe->client->voir && !$socid) $sql.= ", ".MAIN_DB_PREFIX."societe_commerciaux as sc";
 $sql.= " WHERE cf.fk_soc = s.rowid";
-$sql.= " AND s.entity = ".$conf->entity;
+$sql.= " AND s.entity IN (".getEntity("supplier_order").")";
```

Comment:
@mapiolca 
replace with : 
`$sql.= " AND s.entity IN (".getEntity("societe").")";`